personalwebsite/aboutme/models.py

```diff
@@ -2,13 +2,18 @@
 
 # Create your models here.
 class Introduction(models.Model):
-    first_name = models.CharField(max_length=36)
-    last_name = models.CharField(max_length=36)
     profile_image = models.ImageField(upload_to='introduction', null=True, blank=True)
-    description = models.CharField(max_length=256)
+    title = models.CharField(max_length=256)
+
+    def __str__(self):
+        return "Introduction"
+
+class IntroductionContent(models.Model):
+    introduction = models.ForeignKey(Introduction, on_delete=models.CASCADE)
+    paragraph = models.CharField(max_length=512)
 
     def __str__(self):
-        return self.first_name + " - " + self.last_name
+        return self.paragraph
 
 class Section(models.Model):
```

Comment:
I think these two models should just be combined to just **Section** and have a field called **content** inside.

Reply:
That is not possible since a section can have multiple children.

Reply:
Ah, okay. I see how you are using this now and it makes sense.